python/GafferUI/GraphBookmarksUI.py

```diff
@@ -127,6 +144,55 @@ def popupFindBookmarkMenu( editor ) :
 
 	__findBookmark( editor )
 
+def setNumericBookmark( editor, numericBookmark ) :
```

Comment:
set/get are always paired in the Gaffer API, so I suggest `assignNumericBookmark`. Actually, I think I'm leaning more towards a single `GraphBookmarksUI.connect()` method so this is all private, but let's chat about that first - it can be our excuse for a bit of caving...